src/app/application.cpp

```diff
@@ -226,6 +226,7 @@ namespace
 Application::Application(int &argc, char **argv)
     : BaseApplication(argc, argv)
     , m_commandLineArgs(parseCommandLine(Application::arguments()))
+    , m_storeInstanceName(SETTINGS_KEY(u"InstanceName"_s))
```

Comment:
I would still avoid using "instance" term in this matter.
"Instance name" -> "Title suffix"

Reply:
Or did you proceed from the fact that it is enough to avoid "Instance ID", and "Instance Name" does not have a requirement of uniqueness? Hm... maybe there's no problem with it.

Reply:
>Or did you proceed from the fact that it is enough to avoid "Instance ID", and "Instance Name" does not have a requirement of uniqueness? Hm... maybe there's no problem with it.

Yes that is what I thought.

Reply:
> IMO `title` is more correlated to a UI but `Application` may not have a UI. Also 'instance', 'instance name' could be a property of an `Application`.

Okay, I agree. Let it be "instance name". But then why do you still call it "title suffix" in other places? (I mean only those places where it refers exactly to this property. There is no problem in MainWindow class since it is indeed "title suffix" based on that property.)

Reply:
>Okay, I agree. Let it be "instance name". But then why do you still call it "title suffix" in other places? 

I'll rename the option name to "Custom application instance name". The tooltip will stay the same.

Reply:
>I believe "Custom" is redundant since there is no other (default) "instance name" used.

Or 'Customize application instance name'. I want to be clear to users that this is something optional/customizable and not critical to other operations. I hope there won't be strong objections to it.

>Also please rename names in the code to be consistent.

sure.

Reply:
> Or 'Customize application instance name'. I want to be clear to users that this is something optional/customizable and not critical to other operations.

Ok.

Reply:
Done.